torch/csrc/jit/api/method.h

```diff
@@ -32,7 +33,7 @@ struct TORCH_API Method {
       const Kwargs& kwargs = Kwargs());
 
   std::shared_ptr<Graph> graph() const {
-    return function_->graph();
+    return dynamic_cast<FunctionImpl*>(function_)->graph();
```

Comment:
When builtin-functions are added, this can fail. Also we should avoid dynamic casting and use toGraphFunction/toBuiltinFunction methods on Function.